cdc/sinkv2/eventsink/txn/mysql/mysql.go

```diff
@@ -274,71 +274,121 @@ func convertBinaryToString(row *model.RowChangedEvent) {
 }
 
 // TODO: Find a way to make batch delete dmls more efficient.
-func groupRowsByType(
+func (s *mysqlBackend) groupRowsByType(
 	event *eventsink.TxnCallbackableEvent,
 	tableInfo *timodel.TableInfo,
 	spiltUpdate bool,
-) (insertRows, updateRows, deleteRows []*sqlmodel.RowChange) {
+) (insertRows, updateRows, deleteRows [][]*sqlmodel.RowChange) {
```

Comment:
could you add a test case to verify the issue.

Reply:
sure, I will do it.